src/apollo/queries.js

```diff
@@ -56,13 +56,13 @@ export const UNISWAP_PRICES_QUERY = gql`
 `;
 
 export const UNISWAP_ALL_TOKENS = gql`
-  query tokens($excluded: [String]!, $first: Int!, $skip: Int!) {
```

Comment:
We used to exclude fetching the "curated" items from the subgraph. Now we fetch everything and map by the rainbow token list verified flag instead of just using the curated list as the verified items.